plugin/trino-raptor-legacy/src/test/java/io/trino/plugin/raptor/legacy/storage/TestShardRecovery.java

```diff
@@ -160,29 +163,29 @@ public void testShardRecoveryExistingFileChecksumMismatch()
 
         // write data and backup
         File tempFile = createTempFile("tmp", null, temporary);
-        Files.write("test data", tempFile, UTF_8);
+        asCharSink(tempFile, UTF_8).write("test data");
 
         backupStore.backupShard(shardUuid, tempFile);
         assertTrue(backupStore.shardExists(shardUuid));
 
         File backupFile = backupStore.getBackupFile(shardUuid);
-        assertTrue(Files.equal(tempFile, backupFile));
```

Comment:
Don't static import `equal`, since the name is ambiguous without the `File.` prefix